requirements.txt

```diff
@@ -7,6 +7,7 @@ Click==7.0
 PyYAML~=5.3
 eql~=0.9
 elasticsearch~=7.5.1
+XlsxWriter==1.3.6
```

Comment:
is this a heavy dependency? is it worth making it optional?

Reply:
using a separate `docs_requirements.txt`?